sdk/signalr/Microsoft.Azure.WebJobs.Extensions.SignalRService/src/TriggerBindings/ServerlessHubOfT.cs

```diff
@@ -0,0 +1,57 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.IdentityModel.Tokens.Jwt;
+using System.Linq;
+using System.Security.Claims;
+using System.Threading.Tasks;
+using Azure.Core.Pipeline;
+using Microsoft.Azure.SignalR.Management;
+
+namespace Microsoft.Azure.WebJobs.Extensions.SignalRService
+{
+    [SuppressMessage("StyleCop.CSharp.DocumentationRules", "SA1649:File name should match first type name", Justification = "https://github.com/Azure/azure-sdk-for-net/issues/17164")]
+    public abstract class ServerlessHub<T> where T : class
```

Comment:
Is it possible to derive from `ServerlessHub`?  E.g. `public abstract class ServerlessHub<T> : ServerlessHub where T : class`

Reply:
`ServerlessHub` has a property `IHubClients` which `ServerlessHub<T>` doesn't have. The root cause is `IHhubClients<T>` doesn't derive from `IHubClients`.

Reply:
You can use `new` to overwrite it.  https://github.com/dotnet/aspnetcore/blob/48c76f74df8312c47137aeb7a7beddf84599b0ec/src/SignalR/server/Core/src/Hub%60T.cs#L19